packages/snowflake/sodasql/dialects/snowflake_dialect.py

```diff
@@ -56,6 +56,17 @@ def default_connection_properties(self, params: dict):
             'schema': 'PUBLIC'
         }
 
+    def safe_connection_data(self):
+        return [
+            self.type,
+            self.database,
+            self.account,
```

Comment:
we will push this into `datasource_id`

Reply:
I would leave it at `account`

Reply:
Account is unique enough for the purposes, so we can just have account here.

Reply:
removed account